### DISCUSSION TITLE
Cant login to copilot on vscode.

### DISCUSSION THREAD

**Discussion by jb-abacix** [community/community#147446] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have a paid account and I was already logged in. Now with the Free Copilot upgrade to the extension, I get a "Welcome to Copilot" message. I dont want the free version since I have a paid account, but when I click on "Sign in with a GitHub.Com account" nothing happens. If I click "Use copilot for Free" I get this error:

Trilogy::ProtocolError: 1062: Duplicate entry '24683032' for key 'copilot_limited_users.index_copilot_limited_users_on_user_id' (trilogy_query_recv)

This was working perfectly until this recent update.

**Comment by rwkirby** [comment]
Getting a similar error:

Trilogy::ProtocolError: 1062: Duplicate entry '60716810' for key 'copilot_limited_users.index_copilot_limited_users_on_user_id' (trilogy_query_recv)

**Comment by ghuamano** [comment]
Getting a similar error:
Trilogy::ProtocolError: 1062: Duplicate entry '110306045' for key 'copilot_limited_users.index_copilot_limited_users_on_user_id' (trilogy_query_recv)

**Comment by joakim** [comment]
I used to have a trial account that expired. I now want to use Copilot for free and pressed the big button in VSCode.

```
Trilogy::ProtocolError: 1062: Duplicate entry '[redacted]' for key 'copilot_limited_users.index_copilot_limited_users_on_user_id' (trilogy_query_recv)
```

https://github.com/settings/copilot says:

> Your trial of Copilot has ended. Please purchase to continue to use Copilot.

When I press the big "Start using Copilot Free" button, I get the same error as above.

Relevant GitHub Copilot output from VSCode:

```
[info] [certificates] Removed 12 expired certificates
[info] [auth] Invalid copilot token: missing token: 403 
[error] [auth] auth: CopilotAuthError: Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as [redacted].
    at authShowWarnings (/Users/[redacted]/.vscode/extensions/github.copilot-1.253.0/extension/src/auth.ts:114:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at gae.getCopilotToken (/Users/[redacted]/.vscode/extensions/github.copilot-1.253.0/extension/src/auth.ts:144:33) {
  cause: undefined
}
```

**Comment by ghuamano** [comment]
Now, I just started the 30-day free trial and the error is gone, I don't know if it's due to a problem with the new "free plan" only, but now I'll try to see how it goes after these 30-day trial.

**Comment by wasiqs-classics** [comment]
Received the same error

Trilogy::ProtocolError: 1062: Duplicate entry '127827260' for key 'copilot_limited_users.index_copilot_limited_users_on_user_id' (trilogy_query_recv)

**Comment by sebafra** [comment]
Same error:

Trilogy::ProtocolError: 1062: Duplicate entry '1395787' for key 'copilot_limited_users.index_copilot_limited_users_on_user_id' (trilogy_query_recv)

**Comment by PleatherStarfish** [comment]
I'm getting the same error:

`Trilogy::ProtocolError: 1062: Duplicate entry '10040486' for key 'copilot_limited_users.index_copilot_limited_users_on_user_id' (trilogy_query_recv)`

**Comment by marshallwg** [comment]
Also getting the error when I click on the link on the email, and on the link in the MSoft blog announcement. 

`Trilogy::ProtocolError: 1062: Duplicate entry '60490220' for key 'copilot_limited_users.index_copilot_limited_users_on_user_id' (trilogy_query_recv)`

**Comment by pierotticastillo** [comment]
Tengo el mismo problema

**Comment by Akash1134** [comment, marked as answer]
This issue "1062: Duplicate entry" affected complimentary access to GitHub Copilot Pro for students, teachers and maintainers, and resulted into an error. We've fixed it and you can continue enjoying your complimentary access to GitHub Copilot Pro!
